ReactAndroid/src/main/java/com/facebook/react/bridge/ReadableMap.java

```diff
@@ -26,4 +28,6 @@
   Dynamic getDynamic(String name);
   ReadableType getType(String name);
   ReadableMapKeySetIterator keySetIterator();
+  HashMap<String, Object> toHashMap();
```

Comment:
ReadableMap.toMap sounds a bit strange though. I agree with @dcestari that this makes more sense as HashMap.

If both parties agree, I would suggest
List<Object> toList
Map<String, Object> toJavaMap

This would probably deal with the confusion.